docs/book/src/cronjob-tutorial/testdata/project/test/e2e/e2e_test.go

```diff
@@ -108,6 +108,47 @@ var _ = Describe("controller", Ordered, func() {
 			EventuallyWithOffset(1, verifyControllerUp, time.Minute, time.Second).Should(Succeed())
 		})
 
+		It("should provisioned cert-manager", func() {
+			By("validating that cert-manager has the certificate Secret")
+			EventuallyWithOffset(1, func() error {
```

Comment:
Here too, it is (IMHO) easier to read when you send in the `g Gomega` and do regular Gomega assertions rather than typical go _error checking_.  Something like this, perhaps:

```
EventuallyWithOffset(1, func(g Gomega) {
    cmd := ...  
    g.Expect(utils.Run(cmd)).Error().ToNot(HaveOccurred())
})
```